### ISSUE TITLE
Visual Studio 15 hangs as soon as I start typing

### ISSUE THREAD

**Issue by dwenegar** [Dreamescaper/IntelliSenseExtender#8]
Hi. As the titled says, with the latest version of this extension, Visual Studio hangs as soon as I start typing.
Using `git bisect` I found that the problem has been introduced by the commit a01ae1ff100d4fdb874e9c97a6112e8c32c3e10a (Update packages to latest versions).
Reverting that commit, and rebuilding the extensions solved the problem for me.

**Comment by iccfish** [comment]
My vs2017 just crashed with error message `unknown hard error`. Logs in eventvwr just shown as following:

```
应用程序: devenv.exe
Framework 版本: v4.0.30319
说明: 应用程序通过 System.Environment.FailFast(string message) 请求终止进程。
消息: System.IO.FileNotFoundException: 未能加载文件或程序集“System.Collections.Immutable, Version=1.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a”或它的某一个依赖项。系统找不到指定的文件。
文件名:“System.Collections.Immutable, Version=1.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a”
   在 IntelliSenseExtender.IntelliSense.Providers.AbstractCompletionProvider.FilterNamespace(INamespaceSymbol ns)
   在 IntelliSenseExtender.IntelliSense.Providers.AbstractCompletionProvider.<>c__DisplayClass8_0.<<GetAllTypes>g__getAllTypes|0>d.MoveNext()
   在 System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   在 System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 IntelliSenseExtender.IntelliSense.Providers.UnimportedCSharpCompletionProvider.<ProvideCompletionsAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.CodeAnalysis.Completion.CompletionServiceWithProviders.<GetContextAsync>d__31.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.CodeAnalysis.Completion.CompletionServiceWithProviders.<ComputeNonEmptyCompletionContextsAsync>d__24.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   在 Microsoft.CodeAnalysis.Completion.CompletionServiceWithProviders.<GetCompletionsAsync>d__22.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.CodeAnalysis.Completion.CompletionService.<GetCompletionsAndSetItemDocumentAsync>d__11.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.Completion.Controller.Session.ModelComputer.<DoInBackgroundAsync>d__13.MoveNext()
```

After debug I found that this crash due to `IntelliSenseExtender.dll`, so I disabled **IntelliSense Extender** and solved this issue.

cc https://github.com/Microsoft/dotnet-apiport/issues/471

**Comment by Dreamescaper** [comment]
I suppose that happens because you're using not the latest VS version. Correct me if I'm wrong, please.
WIl release hotfix release soon.

**Comment by iccfish** [comment]
@Dreamescaper  sorry for that's not true. My VS2017 was the lastest version, even newer than stable version.
Version: Visual Studio 2017 Preview, 15.6.0 Preview 1.1 (aka VS2017 Update 6 Preview).

**Comment by SirRufo** [comment]
I have the same issue on the current stable (15.5.2) and the current preview (15.6.0 Preview 1.1) of VS2017

**Comment by Dreamescaper** [comment]
Should be fixed now (in 0.6.1) . 
Let me know if it's still there, please.
Thanks a lot!
